src/service/url-expander/expander.js

```diff
@@ -127,25 +130,40 @@ export class Expander {
           builder = '';
         }
 
-        else if (numOfPendingCalls && url[urlIndex] === ',') {
+        else if (url[urlIndex] === PARSER_IGNORE_FLAG) {
+          if (!ignoringChars) {
+            ignoringChars = true;
+            nextArgShouldBeRaw = true;
+            builder = '';
```

Comment:
`user().assert(builder === '')` first. This can hide bugs such as 

```
build`test`
```

missing the `build`

Reply:
this is a good idea. Will make a separate PR since this is now merged